### DISCUSSION TITLE
Rest Query Binding in Transformer

### DISCUSSION THREAD

**Discussion by aloreaux** [Budibase/budibase#7031] [Help] [answered]
I see in the the docs here: https://docs.budibase.com/docs/rest-bindings

that you can use the bindings in the transformer section but I do not see an example of it. I have tried a few ways using just the handlebars and using '$()' that I saw in other docs and that has also failed.

How do you use bindings in the transformer?

Thanks!

**Comment by mike12345567** [comment, marked as answer]
Hi @aloreaux - you are right we should document this properly, right now you can access the parameters that were passed to the query via the `params` object available in context, like so:

![image](https://user-images.githubusercontent.com/4407001/182361796-74c49488-36b0-4ca1-b403-dc20cbaaa81b.png)

I've configured a binding called `binding` and given it a default value of `1` - you can see in the JSON result that it has been returned.

EDIT: I've updated the docs to explain this with an example as well.

**Reply by aloreaux** [reply]
Great! Thank you!
